osmosis/assetlist.json

```diff
@@ -1694,6 +1694,52 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/RAPTR.png"
         }
       ] 
+    },
+    {
+      "description": "GLTO on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+          "exponent": 0
+        },
+        {
+          "denom": "glto",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+      "name": "Gelotto",
+      "display": "glto",
+      "symbol": "GLTO",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "glto",
+            "channel_id": "channel-8"
+          },
+          "chain": {
+            "channel_id": "channel-122",
+            "path": "transfer/channel-122/glto"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "injective",
+            "base_denom": "glto"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/glto.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/glto.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/strd.png",
```

Comment:
There images (point to STRD) are incorrect. If it's going to stay the same, please use an image_sync to Juno GLTO